src/main/java/Task.java

```diff
@@ -0,0 +1,32 @@
+public class Task {
+    protected static String[] tasks = new String[100];
+    protected static boolean[] tasksCompleted = new boolean[100];
```

Comment:
whichTasksCompleted sounds like a better variable name as the variable name tasksCompleted sounds like an integer